src/main/scala/com/amazon/deequ/analyzers/QuantileNonSample.scala

```diff
@@ -42,6 +42,22 @@ class QuantileNonSample[T](
 
   expand()
 
+  /** Given sketchSize, shrinkingFactor, and data, reconstruct the KLL Object */
+  def reconstruct(k: Int, c: Double, data: Array[Array[T]]): Unit = {
```

Comment:
why do we use different names for sketchSize (k), and shrinkingFactor (c)?

Reply:
You are right, it could cause confusions. Fixed.